source/users.rst

```diff
@@ -47,14 +53,17 @@ permissions have been successfully granted.
 `System Users <users.html#system-users>`_
 -------------------------------------------
 
-*Default Secret name:* ``my-cluster-secrets``
+To automate the deployment and management of the cluster components,
+the Operator requires system-level PXC users.
 
-*Secret name field:* ``spec.secretsName``
+Credentials for these users are stored as a `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`_ object.
+The Operator requires to be deployed before the PXC Cluster is started. The name
+of the required secrets (``my-cluster-secrets`` by default) should be set in
+in the ``spec.secretsName`` option of the ``deploy/cr.yaml`` configuration file.
 
-The Operator requires system-level PXC users to automate the PXC
-deployment.
+The following table shows system users' names and purposes.
 
-**Warning:** *These users should not be used to run an application.*
+.. warning:: These users should not be used to run an application.
 
 .. tabularcolumns:: |p{1.5cm}|p{1.5cm}|p{1.5cm}|p{2.5cm}|L|
 
```

Comment:
line 80, root user can be used in applications if needed. Operator doesn't use is anymore internally.
we created `operator` user for this.
we need to add it